conf/messages.conf

```diff
@@ -289,7 +289,8 @@
 287: You cannot change party leaders in this map.
 //Missing stuff for @killer related commands.
 288: You are no longer killable.
-//289-290 FREE
+289: Target character must be in the same map.
+//290 FREE
```

Comment:
The packet (`0x291`) [is in](https://github.com/HerculesWS/Hercules/blob/master/src/map/packets.h#L1028) the `PACKETVER >= 20060424` section though and the comment says `2006-04-24aSakexe`